README.md

```diff
@@ -959,3 +983,37 @@ The wallet-proxy queries the database in specific patterns. Every query filters
 Some queries additionally filter out per transaction type or by time.
 To support efficient retrieval in the common cases it is necessary that the `ati` and `cti` tables have a primary key index on the joint `(account, id)` columns (and analogous columns `(index, subindex, id)` for the `cti` table).
 The order of columns matters, since PostgreSQL will make best use of these indices if the queries have an equality constraint on the leading columns and a relational constraint on the remaining ones.
+
+## Forced update configuration file
+
+If the update configuration file is not present then the `/v0/appSettings`
+endpoint will always return `ok`. If the file is present it must be a valid JSON
+file in the following format
+
+```json
+{
+    "ios": {
+        "forceUpdateVersions": "5,8,10-17",
+        "suggestUpdateVersions": "9",
+        "url": "link to app store"
+    },
+    "android": {
+        "forceUpdateVersions": "-64",
+        "suggestUpdateVersions": "65-80",
+        "url": "link to app store"
+    }
+}
+```
+where both `ios` and `android` are optional and default to empty ranges if not
+present.
+
+The field
+- `forceUpdateVersions` must be a string that denotes ranges of app versions
+  that should be forced to update.
+- `suggestUpdateVersions` must a string that denotes ranges of app versions where
+  the users should upgrade, but are not required to.
+  
+The range format is fairly standard, e.g., `5,8,10-17` denotes app versions `5`,
+`8`, and `10` to `17`, inclusive. Infinite ranges are also supported, e.g.,
+`-64` denotes the range of versions `<= 64`.
```

Comment:
Consider adding a reference to the library used to parse: https://hackage.haskell.org/package/range-0.3.0.2/docs/Data-Range-Parser.html#v:parseRanges

Although I see in this case it doesn't add a lot of details...

Reply:
I think I described the format as well as the library does. I'd prefer not to expose implementation details such as the library used in the README as it is now.